server/test/comment.spec.js

```diff
@@ -0,0 +1,126 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../index';
+
+chai.use(chaiHttp);
+
+let userToken;
+before(async () => {
+  const userResponse = await chai
+    .request(app)
+    .post('/api/v1/auth/login')
+    .send({
+      email: 'mofe@okoro.com',
+      password: 'soldier'
+    });
+  userToken = userResponse.body.data.token;
```

Comment:
I think maybe destructuring the token from the `body` would be ES6 compliant

Reply:
Ok would look into that now, thanks